GoogleForCreators/web-stories-wp
# PPC 2.0: Implement new panels in checklist

## Context

PPC 2.0 
[Designs (components)](https://www.figma.com/file/bMhG3KyrJF8vIAODgmbeqT/Design-System?node-id=8393%3A38)
[Designs (final delivery)](https://www.figma.com/file/1YM3jA9h4QGc4xLdyaXIJS/Stories-Sprint-1.7?node-id=8378%3A3778)

## Summary

- Add the tablist panels to the checklist
- Add the empty state to the checklist
- Add karma tests to wrap it up in a bow

## Relevant Technical Choices

- some checks weren't hooked into the `useRegisterCount` hook, so added those here.

## To-do


- [ ] Add karma tests to clear out checklist by fixing issues (future pr)


## User-facing changes

|With Issues|Empty State|
|--|--|
|![checklist](https://user-images.githubusercontent.com/22185279/123706594-b19a4880-d825-11eb-8a67-0fb9720b51dd.gif)|![image](https://user-images.githubusercontent.com/22185279/123486915-3426cc00-d5ca-11eb-8e2c-1d1ca7ff8f05.png)|

## Testing Instructions

1. Enable ppc 2.0 experiment
2. Create new story
3. look at the prettiness! Also, tests should pass

### QA






## Reviews

### Does this PR have a security-related impact?

no

### Does this PR change what data or activity we track or use?

no

### Does this PR have a legal-related impact?

no

## Checklist




---



Fixes #8084
